libpkl/gradle.lockfile

```diff
@@ -59,20 +59,26 @@ org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.6.4=kotlinBuildToolsApiClass
 org.jetbrains:annotations:13.0=compileClasspath,kotlinBuildToolsApiClasspath,kotlinCompilerClasspath,kotlinCompilerPluginClasspathMain,kotlinCompilerPluginClasspathNativeTest,kotlinCompilerPluginClasspathTest,kotlinCompilerPluginClasspathTestJdk17,kotlinKlibCommonizerClasspath,nativeTestCompileClasspath,nativeTestImplementation,nativeTestRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
 org.jetbrains:markdown-jvm:0.7.3=nativeTestImplementation,nativeTestRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
 org.jetbrains:markdown:0.7.3=nativeTestImplementation,nativeTestRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
-org.junit.jupiter:junit-jupiter-api:5.12.2=nativeTestCompileClasspath,nativeTestImplementation,nativeTestImplementationDependenciesMetadata,nativeTestRuntimeClasspath,testCompileClasspath,testImplementationDependenciesMetadata,testRuntimeClasspath
+org.junit.jupiter:junit-jupiter-api:5.12.2=nativeTestImplementation
+org.junit.jupiter:junit-jupiter-api:5.13.0=nativeTestCompileClasspath,nativeTestImplementationDependenciesMetadata,nativeTestRuntimeClasspath,testCompileClasspath,testImplementationDependenciesMetadata,testRuntimeClasspath
 org.junit.jupiter:junit-jupiter-api:5.8.2=testJdk17CompileClasspath,testJdk17ImplementationDependenciesMetadata,testJdk17RuntimeClasspath
-org.junit.jupiter:junit-jupiter-engine:5.12.2=nativeTestCompileClasspath,nativeTestImplementation,nativeTestImplementationDependenciesMetadata,nativeTestRuntimeClasspath,testRuntimeClasspath
+org.junit.jupiter:junit-jupiter-engine:5.12.2=nativeTestImplementation
+org.junit.jupiter:junit-jupiter-engine:5.13.0=nativeTestCompileClasspath,nativeTestImplementationDependenciesMetadata,nativeTestRuntimeClasspath,testRuntimeClasspath
```

Comment:
We now have three versions in use: 5.8.2, 5.12.2, and 5.13.0. Can we clean this up?